src/games/seabattle/sound.test.ts

```diff
@@ -31,14 +37,9 @@ test('should not play sound if it is unrelated update - online friend', () => {
       },
     ],
   };
-  expect(getSound(gameArgsOnline, undefined, action)).to.equal(null);
-  SeabattleSound(gameArgsOnline)(mockStore)(jest.fn())(action);
+  SeabattleSound('HIT');
```

Comment:
What tryngi to test? why do you still have the action? what are you expecting from this test?

Reply:
![giphy](https://user-images.githubusercontent.com/304383/57590491-27ce7a80-74fa-11e9-95ec-6604b998da02.gif)